loader-hook.mjs

```diff
@@ -0,0 +1 @@
+export * from 'import-in-the-middle/hook.mjs'
```

Comment:
Any way to not have to specify the file directly? Something like `--loader=dd-trace` or `--loader=dd-trace/hook`.

Reply:
Try having an index.mjs inside a hook folder, so we can spare the .mjs